src/java.base/share/classes/sun/security/pkcs/PKCS9Attributes.java

```diff
@@ -325,12 +304,10 @@ public String toString() {
         PKCS9Attribute value;
 
         boolean first = true;
```

Comment:
While you are changing this method, add an `@Override` annotation to `toString`.